docs/source/notes/extending.rst

```diff
@@ -862,10 +917,3 @@ See this
 for a detailed explanation and examples.
 
 Documentations are available at :doc:`../cpp_extension`.
-
-
-Writing custom C extensions
----------------------------
-
-Example available at
-`this GitHub repository <https://github.com/pytorch/extension-ffi>`_.
```

Comment:
is this just outdated? or is the removal unintentional

Reply:
Yes this repo has been dead for a while. I archived it as well.